addons/binding/org.openhab.binding.rotelra1x/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="rotelra1x"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <thing-type id="amp">
+        <label>Rotel Amplifier</label>
+        <description>Rotel RA11 / RA12 stereo amplifier.</description>
+
+        <channels>
+            <channel id="power" typeId="power"/>
+            <channel id="volume" typeId="volume"/>
+            <channel id="mute" typeId="mute"/>
+            <channel id="source" typeId="source"/>
+            <channel id="frequency" typeId="frequency"/>
+            <channel id="dimmer" typeId="dimmer"/>
+        </channels>
+        
+        <config-description>
+            <parameter name="port" type="text">
+                <label>Serial port</label>
+                <description>Select serial port (COM1, /dev/ttyS0, ...)</description>
+                <default>COM1</default>
+                <required>true</required>
+            </parameter>
+            <!-- Max volume could have been queried using the protocol, but would have 
+                 complicated the code, and it could be nice to set 100 % at less than
+                 the receiver's max. -->
+            <parameter name="maximum-volume" type="integer">
```

Comment:
Maybe configure a minimum of 1, otherwise you will have division by zero errors.